components/x-gift-article/src/RadioButtonsSection.jsx

```diff
@@ -63,5 +57,54 @@ export default ({
 			/>
 			<span className="o-forms-input__label">FT subscribers only</span>
 		</label>
-	</div>
-)
+	)
+
+	const freeToReadField = () => (
+		<label htmlFor="nonGiftLink">
+			<input
+				type="radio"
+				name="gift-form__radio"
+				value="nonGiftLink"
+				id="nonGiftLink"
+				checked={shareType === ShareType.nonGift}
+				onChange={showNonGiftUrlSection}
+			/>
+			<span className="o-forms-input__label">with anyone</span>
+		</label>
+	)
+
+	if (isFreeArticle) {
+		if (enterpriseEnabled) {
+			return (
+				<div
+					className="o-forms-input o-forms-input--radio-round o-forms-field x-gift-article__radio_buttons"
+					role="group"
+					aria-labelledby="article-share-options"
+				>
+					<span className="x-gift-article--visually-hidden" id="article-share-options">
+						Article share options
+					</span>
```

Comment:
Suggestion:
All this part is duplicated in [line97](https://github.com/Financial-Times/x-dash/pull/684/files#diff-4b3d233b646176226bcb2b4b467761b8775262f3b727dfdf7254d39b33b44c08R97). You can structure the composition in a way that avoids duplications.

Duplications, in this case, will just make it hard to refactor this component where needed.